perfkitbenchmarker/data/__init__.py

```diff
@@ -28,9 +28,10 @@
 
 import pkg_resources
 
+from perfkitbenchmarker import context_util
 from perfkitbenchmarker import flags
 
-FLAGS = flags.FLAGS
+FLAGS = context_util.FLAGS
```

Comment:
I'm a little concerned about references to `flags.FLAGS` creeping in elsewhere. Is there any way to make a single `FLAGS` import location?